tools/extra/pac/fpgabist/bist_common.py

```diff
@@ -35,8 +35,7 @@
 BIST_MODES = ['bist_afu', 'dma_afu', 'nlb_mode_3']
 REQ_CMDS = ['lspci', 'fpgainfo', 'fpgaconf', 'fpgadiag', 'fpga_dma_test',
             'fpga_dma_vc_test', 'bist_app']
-BDF_PATTERN = (r'\d+:(?P<bus>[a-fA-F0-9]{2}):'
-               r'(?P<device>[a-fA-F0-9]{2})\.(?P<function>[a-fA-F0-9])')
+BDF_PATTERN = r'{:02x}:{:02x}.{:x}'
```

Comment:
what is the motivation changing this BDF Pattern.
It seems fine but would need to see tests passed. Was there an issue finding the appropriate host?
I'd also like to see how this works when using an VM to run the fpgabist tool

Reply:
@buie1 I changed the pattern because you asked me to in the previous review.